pkg/asset/ignition/bootstrap/bootstrap_in_place.go

```diff
@@ -0,0 +1,21 @@
+package bootstrap
+
+import (
+	"github.com/pkg/errors"
+
+	"github.com/openshift/installer/pkg/types"
+)
+
+// verifyBootstrapInPlace validate the number of control plane replica is one and that installation disk is set
+func verifyBootstrapInPlace(installConfig *types.InstallConfig) error {
+	if *installConfig.ControlPlane.Replicas != 1 {
+		return errors.Errorf("bootstrap in place require single control plane replica, current value: %d", *installConfig.ControlPlane.Replicas)
```

Comment:
nit: "bootstrap in place requires a single control plane replica" ("require single" -> "requires a single")